letsencrypt/client/constants.py

```diff
@@ -44,6 +44,18 @@
 """Port to perform DVSNI challenge."""
 
 
+DNS_CHALLENGE_TTL = 60
+"""TTL, in seconds, for the _acme-challenge.domain TXT record."""
+
+
+DNS_CHALLENGE_SOURCE_PORT = 0
+"""Port to issue DNS queries from."""
+
+
```

Comment:
nit: reduce whitespace by one line

the one between `DNS_CHALLENGE_TIMEOUT` and `TEMP_CHECKPOINT_DIR` is correct